### ISSUE TITLE
Crashes on launch when trying to use texture pack on OSX/Linux

### ISSUE THREAD

**Issue by gamrguy** [blushiemagic/tModLoader#213]
### Description

When attempting to run Terraria with a texture pack (Texture Pack.zip in the TexturePacks folder) the game will immediately crash on launch. This only occurs on OSX/Linux - users on Windows appear to be fine.

### Steps to Reproduce

0. Have Experimental Features enabled (to allow loading texture packs)
1. Place a Texture Pack.zip in the ModLoader/TexturePacks folder.
2. Attempt to run Terraria.
3. See that the game crashes.

### Expected behavior 

...that the game wouldn't crash and would load the texture pack as normal. Or perhaps multiple texture packs, as there is a "TexturePacks" folder for a reason, eh? Is multiple texture packs a feature?

### Actual behavior

As stated: the game crashes.

### Reproduction frequency

100%

### Versions

I once tested this a while back (might've been a 0.8 version, not sure) but was tested for tModLoader v0.10.0.2.

### Mods

The crash occurs before the game starts loading properly. No mods are involved.

### Additional Information

The crash occurs without the game window ever opening; on OSX, the app icon briefly appears on the dock and quickly closes. Running the game via Terminal reveals no crashlog.

**Comment by Jofairden** [comment]
Can you check ModLoader/Logs ? Anything specific in there?

**Comment by gamrguy** [comment]
I checked the logs. None were updated upon attempting to launch the game.

**Comment by TheOnlyMrCat** [comment]
Other people with this issue have reported a log looking like this:

```
System.TypeInitializationException: The type initializer for 'Ionic.Zip.ZipFile' threw an exception. ---> System.NotSupportedException: Encoding 437 data could not be found. Make sure you have correct international codeset assembly installed and enabled.

at System.Text.Encoding.GetEncoding (System.Int32 codepage) [0x0025f] in <1170fb08300b4d0db71c1d1db8555768>:0

at System.Text.Encoding.GetEncoding (System.String name) [0x00015] in <1170fb08300b4d0db71c1d1db8555768>:0

at Ionic.Zip.ZipFile..cctor () [0x00000] in <3d63c7f29fa146c29caee29323a47c7a>:0 --- End of inner exception stack trace ---

at Terraria.TexturePackSupport.FindTexturePack () [0x00029] in <37fa5c86f4774cd0993f32839325bdc4>:0

at Terraria.Main.LoadContent () [0x00099] in <37fa5c86f4774cd0993f32839325bdc4>:0

at Microsoft.Xna.Framework.Game.Initialize () [0x00053] in <bc6b38abaaf04135afbd0534ec7f6b92>:0

at Terraria.Main.ClientInitialize () [0x00082] in <37fa5c86f4774cd0993f32839325bdc4>:0

at Terraria.Main.Initialize () [0x0640a] in <37fa5c86f4774cd0993f32839325bdc4>:0

at Microsoft.Xna.Framework.Game.DoInitialize () [0x0000d] in <bc6b38abaaf04135afbd0534ec7f6b92>:0

at Microsoft.Xna.Framework.Game.Run () [0x0000e] in <bc6b38abaaf04135afbd0534ec7f6b92>:0

at Terraria.Program.LaunchGame (System.String[] args, System.Boolean monoArgs) [0x00092] in <37fa5c86f4774cd0993f32839325bdc4>:0
```

**Comment by Z3r0Grav** [comment]
I have the same problem even with normal Terraria on Linux it just does not open with texture packs

**Comment by Chicken-Bones** [comment]
Fixed! Try it out in 0.11 (soon to be released)
